src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
```

Comment:
- [ ] **[REQUIRED]** Kindly change the `Calculator` component from a function-based component to a class-based component.

- [ ] **[OPTIONAL]** Kindly reduce the size of your calculator so that it doesn't cover the entire screen.